README.md

```diff
@@ -9,95 +9,67 @@
 
 ## Install
 
-`adonis install @rocketseat/adonis-bull`
+You need [install Redis Provider](https://preview.adonisjs.com/guides/database/redis/#setup) before install this package.
```

Comment:
Typo: ...before install this package > ...before **installing** this package.

Reply:
Fixed, thanks! 😊